TenancyInformationApi/V1/Infrastructure/UhContext.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace TenancyInformationApi.V1.Infrastructure
+{
+    public class UhContext : DbContext
+    {
+        public UhContext(DbContextOptions options) : base(options)
+        {
+            var tenancyAgreements = UhTenancyAgreements
+                .Include(t => t.UhAgreementType)
+                .Include(t => t.UhTenureType)
+                .ToList();
```

Comment:
I'd forgotten about this, I expect it will cause issues with timeouts when testing. But I guess it can be dealt with when we come to it!